### ISSUE TITLE
Welcome to the JavaScript Hall of Fame!

### ISSUE THREAD

**Issue by michaelrambeau** [HugoGiraudel/ama#63]
Hello Hugo, you have been inducted to the JavaScript Hall of Fame http://bestof.js.org/hof, I think you are the only French, congratulations!

I created bestof.js.org to gather the best open source projects related to the web platform.
Every day the application checks stars on Github for a curated list of about 500 projects.
The application contains one project from you: [milligram](http://milligram.github.io/) (what a nice CSS toolkit!).

In a way your [SJSJ](http://jargon.js.org/) project is an attempt to solve the same problem as bestof.js.org: a place to find curated content about this JS world that changes everyday...
Any feedback from you would be very appreciated.
Sorry, it was not really a question, it was more a feedback request, thank you!
Merci, à bientôt !

![image](https://cloud.githubusercontent.com/assets/5546996/15444159/cdfac2e0-1f28-11e6-8109-c879ca77aba4.png)

**Comment by HugoGiraudel** [comment]
Hey there,

Congratulations on bestof.js.org, it looks good. I dig what you did there. I have been browsing a bit and can only assume it was interesting to build.

---

Now, I am really not sure who and how anyone would get a real benefit from it. There are tons of information, for sure, but nothing here is extremely relevant. Number of stars, of published packages… They are a lot of metrics that are fun to play around but does not necessarily reflect anything.

By no mean do I intend to say this project is not worth it, though. I do like it, and I find also nice to browse, but I am not sure I understand your goal behind it. Was it just a play project or do you aim at something specific?

The problem, I believe, is that we don’t (or rather shouldn’t) pick packages, libraries or tools for their fame. Pick the right tool for the right job, it doesn’t matter if it has 50 stars or 25000. Therefore, promoting everything based on fame metrics makes it hard for me to see the real value.

Now, I am not sure how to fix that. Maybe hand-picking interesting projects and expanding a bit on them could be interesting? Or having comparisons between similar projects (Underscore / Lodash / Ramda comes to my mind)? I don’t know.

--- 

On the design side, I think it works pretty well. I like the brownish scheme and I believe the whole thing is pretty okay. My main problem comes from the abundance of information, making it hard to read. It’s like the cockpit of a plane, you know; with all these buttons everywhere, only an experienced used would find their way out. Maybe it’s just about having a better visual hierarchy of information. Making meaningful information stand out more than secondary information (through size, color…).

---

On the technical side, the only downside I can find is the lack of real URLs. I don’t see any good reason to hash-bang everything, so I would encourage you to move to actual URLs. I believe it would be much better.

---

Here are my 2 cents.
Hope it helps. :)

-Hugo

**Comment by michaelrambeau** [comment]
Hello Hugo, thank you very much for your feedback, that's really nice from you!
I get your point, sometimes a concept works, sometimes it does not, I will think about what you said, especially about showing relevant information.

My goal was to create a tool that shows only curated projects (all projects were hand picked) and that was super-fast, a kind of fast search engine through bookmarked projects.
For example, if I enter "flex", I wanted to find right away "Solved by Flexbox" project, without having to google it.

URLs are hash-banged because Github pages does not support "Client-side Routing"  like netlify.com does for example, and `js.org` domain works only with Github Pages, so I didn't have so much choice. But URLs are part of the UX, I will think about that too (I could generate a static html page every day for all projects for example).

Thank you again!

**Comment by michaelrambeau** [comment]
Hello Hugo,
Just to tell you that I took into account your invaluable feedback about http://bestof.js.org/.
* user-friendly URLs: no more ugly ids
* show **relevant** information: I added information about the **quality** of the packages, using data from http://packagequality.com/ and https://npms.io/
* "airplane cockpit" effect: I tried as much as possible to decrease the amount of things displayed, especially in the top page page
* **interactivity** with the community: logged-in users can now submit Github projects!

![image](https://cloud.githubusercontent.com/assets/5546996/16874115/77d362a2-4ad2-11e6-985c-6ee4c0e24bed.png)

I use the application as super fast search engine when I need to look up a given project.
Someone else told me he use it to be kept up-to-date about the trends because things "evolve so fast” in the JavaScript scene.
So I hope that you will see that the project can be valuable for the community. Anyway it's perfectly fine if you do not ;)

FYI I am thinking about adding project logos or organisation avatars to make the pages more “visual” (and less “airplane-cockpit-like” !).
Feel free to tell me what you think because every time I work on the project, I think “What could I do to make Hugo enjoy the application?", this is my quest!
Thank you again!
